modules/text_server_adv/text_server_adv.h

```diff
@@ -703,16 +703,16 @@ class TextServerAdvanced : public TextServerExtension {
 		int current_priority = 0;
 		uint32_t current_index = 0;
 		uint32_t font_count = 0;
-		const String *language;
-		const String *script_code;
```

Comment:
Note: original strings can get out of scope, so pointer won't work. This might be an issue without this PR as well.